src/main/scala/cromwell/binding/Scatter.scala

```diff
@@ -17,8 +17,12 @@ object Scatter {
  * @param index Index of the scatter block. The index is computed during tree generation to reflect wdl scatter blocks structure.
  * @param item Item which this block is scattering over
  * @param collection Wdl Expression corresponding to the collection this scatter is looping through
+ * @param parent Parent of this scatter
  */
-case class Scatter(index: Int, item: String, collection: WdlExpression, parent: Option[Scope]) extends Scope {
+case class Scatter(index: Int,
+                   item: String,
+                   collection: WdlExpression,
+                   override val parent: Option[Scope]) extends Scope {
```

Comment:
I'm someone who loves me some bells and whistles (or belts and suspenders as @mcovarr would say) but I don't see the value here